scripts/create_cita_config.py

```diff
@@ -400,39 +340,40 @@ def run_subcmd_create(args, work_dir):
         os.path.join(work_dir, 'scripts/config_tool/config_example'))
     info.create_init_data(args.super_admin, args.contract_arguments)
     info.create_genesis(args.timestamp, args.resource_dir)
-    info.encrypted_create_rootca(args.enable_tls)
+    info.enable_tls = args.enable_tls
     for node in args.nodes:
         info.append_node(node)
 
 
 def run_subcmd_append(args, work_dir):
     info = ChainInfo(args.chain_name, work_dir)
     info.template_load_from_existed()
-    info.encrypted_load_from_existed()
     info.append_node(args.node)
 
 
 def parse_arguments():
     from create_init_data import KeyKeyValueDict, KeyKeyValueAction
 
     parser = argparse.ArgumentParser()
-    subparsers = parser.add_subparsers(
-        dest='subcmd', title='subcommands', help='additional help')
+    subparsers = parser.add_subparsers(dest='subcmd',
+                                       title='subcommands',
+                                       help='additional help')
 
     #
     # Subcommand: create
     #
 
-    pcreate = subparsers.add_parser(
-        SUBCMD_CREATE, help='create a chain with nodes')
+    pcreate = subparsers.add_parser(SUBCMD_CREATE,
```

Comment:
auto-fmt?pylint?

Reply:
I use command:
```shell
yapf -i create_cita_config.py
```
and yapf version is `0.27.0`.

Am I make something wrong?

Reply:
I see.
Why use `yapf`? and it should be another commit if you format the code.

Reply:
Oh, `pylint` is okay!

Reply:
> For instance, code that conforms to the PEP 8 guidelines may not be reformatted. But it doesn't mean that the code looks good.

> It's based off of 'clang-format'